src/lib/aws-translators.coffee

```diff
@@ -125,7 +126,10 @@ module.exports.scan = (params, options, callback, keySchema) ->
 
   @parent.dynamo.scanAsync(awsParams)
     .then (data)->
-      dataTrans.fromDynamo(data.Items)
+      result =
+        Items: dataTrans.fromDynamo(data.Items),
+        Count: data.Count,
+        LastEvaluatedKey: data.LastEvaluatedKey
```

Comment:
Haven't merged this yet because this signature is not in line with the rest of Dynasty.

I try to abstract the internals of Dynamo from the end user of Dynasty. Hence why all of the other API calls return objects with slightly different names and without capitalizing everything.

Also, this introduces a non-trivial breaking change to Scan. It used to return just an array of objects, now this new object with multiple keys.

Anyway, thanks for the changes! I will do my best to work with them and "Dynasty-ify" them before merging.